load-test/scripts/configuration.js

```diff
@@ -0,0 +1,52 @@
+export const basicTestOptions = {
+  scenarios: {
+    // ramp up to 5 VUs over 30 seconds
+    rampUp: {
+      executor: "ramping-vus",
+      startVUs: 0,
+      stages: [
+        {duration: '30s', target: 5}
+      ]
+    },
+    // have 5 VUs do 3 iterations each, up to a max of 2 minutes. Start this after
+    // the ramp up time.
+    highLoad: {
+      executor: "per-vu-iterations",
+      vus: 5,
+      iterations: 3,
+      maxDuration: "2m",
+      startTime: '30s',
+    }
+  },
+  thresholds: {
+    exceptions: ["count == 0"],
+    errors: ["count == 0"],
+    websocket_session_duration_without_sleep: ["p(95) < 5000"],
+    long_websocket_sessions: ["count <= 1"],
```

Comment:
I'm guessing the count <= 1 is to account for cold boot time right? Is there a separate threshold for that we're also tracking, or not part of this test?

Reply:
yes, there is a threshold for 'extra long websocket sessions'. These are not final thresholds yet, we'll probably need to fine tune to account for the fact that each concurrent instance has its own cold boot time.